README.md

```diff
@@ -153,6 +153,49 @@ client = instructor.from_gemini(
 )
 ```
 
+Alternatively, you can [call Gemini from the OpenAI client](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/call-gemini-using-openai-library#python).You'll have to setup [`gcloud`](https://cloud.google.com/docs/authentication/provide-credentials-adc#local-dev), get setup on Vertex AI, and install the Google Auth library.
+
+```sh
+pip install google-auth
+```
+
+```python
+import google.auth
+import instructor
+from openai import OpenAI
+
+creds, project = google.auth.default()
+auth_req = google.auth.transport.requests.Request()
+creds.refresh(auth_req)
```

Comment:
```suggestion
from google.auth.transport.requests import Request
from openai import OpenAI
from pydantic import BaseModel

creds, project = google.auth.default()
auth_req = Request()
```
Otherwise I get the error:
```
AttributeError: module 'google.auth.transport' has no attribute 'requests'. Did you mean: 'Request'?
```